portfolio.html

```diff
@@ -82,5 +82,237 @@ <h2>I'm Amrendra Glad to see you!</h2>
+          alt="Multipostimage"
+        />
+        <div class="cardheading">Multi-Post Stories</div>
+        <div class="canopyitemwrap">
+          <div class="cardcanopy">CANOPY</div>
+          <div class="dot cardlistitem">&bull;</div>
+          <div class="cardlistitem">Back End Dev</div>
+          <div class="dot cardlistitem">&bull;</div>
+          <div class="cardlistitem">2015</div>
+        </div>
+        <div class="cardabout">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </div>
+        <div class="cardtechnology">
+          <div class="technology">html</div>
+          <div class="technology">css</div>
+          <div class="technology">javaScript</div>
+        </div>
+        <button type="button" class="btnseeproject">See Project</button>
+      </div>
+      <div class="card">
+        <img
+          class="cardimage"
+          src="./Snapshoot_Portfolio_Yoga.png"
+          alt="Yogaimage"
+        />
+        <div class="cardheading">Tonic</div>
+        <div class="canopyitemwrap">
+          <div class="cardcanopy">CANOPY</div>
+          <div class="dot cardlistitem">&bull;</div>
+          <div class="cardlistitem">Back End Dev</div>
+          <div class="dot cardlistitem">&bull;</div>
+          <div class="cardlistitem">2015</div>
+        </div>
+        <div class="cardabout">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </div>
+        <div class="cardtechnology">
+          <div class="technology">html</div>
+          <div class="technology">css</div>
+          <div class="technology">javaScript</div>
+        </div>
+        <button type="button" class="btnseeproject">See Project</button>
+      </div>
+      <div class="card">
+        <img
+          class="cardimage"
+          src="./Snapshoot_Portfolio_MultiPostStories.png"
+          alt="multipoststoryimage"
+        />
+        <div class="cardheading">Multi-Post Stories</div>
+        <div class="canopyitemwrap">
+          <div class="cardcanopy">CANOPY</div>
+          <div class="dot cardlistitem">&bull;</div>
+          <div class="cardlistitem">Back End Dev</div>
+          <div class="dot cardlistitem">&bull;</div>
+          <div class="cardlistitem">2015</div>
+        </div>
+        <div class="cardabout">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </div>
+        <div class="cardtechnology">
+          <div class="technology">html</div>
+          <div class="technology">css</div>
+          <div class="technology">javaScript</div>
+        </div>
+        <button type="button" class="btnseeproject">See Project</button>
+      </div>
+    </section>
+    <!-- ////////// Abmout me Section-->
+
+    <section class="about-myselfsection">
+      <div class="about-myself">About Myself</div>
```

Comment:
- [ ] Please, consider that the title "About Myself" uses a header tag `hx` (aka h1, h2, etc...).